pkg/windows/Dockerfile

```diff
@@ -90,32 +90,21 @@ RUN Set-ExecutionPolicy Bypass -Scope Process -Force; \
 RUN choco install git nssm -y;
 RUN choco install osquery --version 3.4.0 -y;
 
-#RUN powershell.exe $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
-#All the variables used for salt
-ENV SALT_SRC_PATH='C:/temp/salt/'
-ENV SALT_GIT_URL=https://github.com/saltstack/salt
-ENV SALT_CHECKOUT=v2018.11
-#Git clone salt and run packages
-RUN git clone "$env:SALT_GIT_URL"; \
-  Push-Location salt/pkg/windows; \
-  git checkout "$env:SALT_CHECKOUT"; \
-  [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]'Tls,Tls11,Tls12'; \
-  ./build_env_3.ps1 -Silent;
-#more salt installs
-RUN Push-Location salt; \
-  test-path ./setup.py; \
-  python ./setup.py --quiet install --force; \
-  pop-location;
 #All the variables used for hubble
 ARG HUBBLE_CHECKOUT=v4.0.0
 ARG HUBBLE_GIT_URL=https://github.com/hubblestack/hubble.git
 ENV HUBBLE_SRC_PATH='C:/temp/hubble/'
 #Clone Hubble
+
 RUN git clone "$env:HUBBLE_GIT_URL"; \
   Push-Location hubble; \
   git checkout "$env:HUBBLE_CHECKOUT"; \
   pop-location;
 
+RUN Push-Location C:/temp/hubble/pkg/windows; \
+		[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]'Tls,Tls11,Tls12'; \
+		./salt_build_script.ps1 -Silent;
```

Comment:
Do we need this now?

Reply:
This script is needed for installation of windows C++ tools